### PR TITLE
remove/rename/reorder activities, remove coming soon section

### LINKED ISSUE #1695: Edits to /activities
- remove "(Teaching Kit") from everywhere it exists. They are all Teaching Kits, so it's confusing that some say it and others don't.
- Move "Protect Your Data" so that it's directly above the two Privacy Basics kits
- Rename that one "Privacy Basics: Protect Your Data"
- Rename "Back to School Write the Web Kit" to "Back to School Write the Web Activities"
- Remove Become a Password Pro and Hack Your Notebook, since they are covered elsewhere
- Remove Coming Soon section entirely
- Change all references from Mozilla Learning Networks to Mozilla Learning

### PR DESCRIPTION
**Pull request by hannahkane**
fixes #1695 

@mmmavis - for your review, please

### CODE REVIEW

**Comment by Pomax** [comment]
does this address, or resolve 1695? (the phrasing "fixes #..." makes github autoclose the associated ticket on merge, "addresses #..." does not; I've been using that when it's only a partial solution and needs the crosslink)

**Comment by hannahkane** [comment]
thanks, @Pomax. Didn't realize that. I changed it to fixes because it actually resolves the entire issue.

**Comment by mmmavis** [comment]
@hannahkane should we remove "teaching kit" from https://teach.mozilla.org/activities/back-to-school-write-the-web/ as well?

File: pages/activities/ActivitiesPage.jsx [hunk 1]
```diff
@@ -131,22 +131,6 @@ var ActivitiesPage = React.createClass({
               developedBy={<div><a href="http://cogdog.info">Alan Levine</a></div>}
               description="In this activity, learners search for photos that communicate ideas and concepts, while also exploring best practices around attribution and openly-licensed content on the web."/>
             <ActivityKit
-              src1x="/img/pages/activities/img-activity-06.jpg"
-              src2x="/img/pages/activities/img-activity-06@2x.jpg"
```
Comment (mmmavis) [inline pages/activities/ActivitiesPage.jsx]: don't forget to remove image files :wink:

File: pages/activities/ActivitiesPage.jsx [hunk 2]
```diff
@@ -131,22 +131,6 @@ var ActivitiesPage = React.createClass({
...
-              title="Become a Password Pro (Teaching Kit)"
-              level="Beginner-Intermediate"
-              link="https://stacy.makes.org/thimble/MjAxMjIxNzYwMA==/whats-wrong-with-your-password"
-              developedBy={<div><a href="https://mozillians.org/en-US/u/stacy">Stacy Martin</a> and <a href="https://webmaker.org">Mozilla</a></div>}
-              description="This teaching kit includes a series of activities to help leaners understand passwords &mdash; including why they're important, best practices, and tools to help you manage multiple passwords."/>
-            <ActivityKit
-              src1x="/img/pages/activities/img-activity-07.jpg"
-              src2x="/img/pages/activities/img-activity-07@2x.jpg"
```
Comment (mmmavis) [inline pages/activities/ActivitiesPage.jsx]: we should remove these files too

File: pages/activities/ActivitiesPage.jsx [hunk 3]
```diff
@@ -49,31 +49,31 @@ var ActivitiesPage = React.createClass({
...
-              description="These six hands-on activities engage learners in thinking critically about online privacy by creating secure passwords, understanding how and where their data is being collected, and more. This is an ideal kit for Mozilla Clubs."/>
-            <ActivityKit
               src1x="/img/pages/webmaker/designing-webmaker.jpg"
               src2x="/img/pages/webmaker/designing-webmaker@2x.jpg"
               title="Read, Write, and Participate with Webmaker"
               level="Beginner mobile users"
               link="/activities/webmaker/"
-              developedBy={<div><a href="https://twitter.com/secretrobotron">Bobby Richter</a> and <a href="https://twitter.com/lau_nk">Laura de Reynal</a> for Mozilla Learning Networks</div>}
+              developedBy={<div><a href="https://twitter.com/secretrobotron">Bobby Richter</a> and <a href="https://twitter.com/lau_nk">Laura de Reynal</a> for Mozilla Learning</div>}
               description="Learn how to set up your mobile device to write, publish, and share stories with the Webmaker App from Mozilla Learning Networks."/>
```
Comment (mmmavis) [inline pages/activities/ActivitiesPage.jsx]: Mozilla Learning ~~Networks~~ ?

**Comment by hannahkane** [comment]
OK, @mmmavis - Travis is happy with the changes I made.

**Comment by mmmavis** [comment]
+1 merging!

**Pull request merged**